pkg/feature/raw_resources.go

```diff
@@ -105,25 +94,16 @@ func (f *Feature) patchResourceFromFile(filename string) error {
 			Resource: strings.ToLower(u.GroupVersionKind().Kind) + "s",
 		}
 
-		// Convert the patch from YAML to JSON
-		patchAsJSON, err := yaml.YAMLToJSON(data)
+		// Convert the individual resource patch from YAML to JSON
```

Comment:
I believe it's necessary as we want to convert to JSON in case of JSON merge patch or JSON patch formats used for patching - @bartoszmajsak can you confirm or am I off base here?

Reply:
you are right
by using `MergePatchType` it uses "application/merge-patch+json" which need to be json format.